css/amp.css

```diff
@@ -219,10 +219,10 @@ i-amphtml-sizer {
  * does not allow activation of buttons.
  */
 .i-amphtml-screen-reader {
-  position: fixed !important;
-  /* keep it on viewport */
-  top: 0px !important;
-  left: 0px !important;
+  position: absolute !important;
+  /* keep it on viewport. TalkBack likes this >=12px */
+  top: 12px !important;
```

Comment:
As long as they are not accidentally triggerable by tap, we should be good. Could you do a bit of testing on that? Just do lots of taps around the area where they are now.